modules/checkout/commerce_checkout.module

```diff
@@ -32,6 +33,20 @@ function commerce_checkout_theme() {
 }
 
 /**
+ * Implements hook_help().
+ */
+function commerce_checkout_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'entity.commerce_checkout_flow.collection':
+      $output = '<p>' . t('Checkout flows are used by <a href=":order_types">order types</a> to determine the order and appearance of checkout panes including payment, contact, and shipping information.', [
```

Comment:
We should say "billing information" instead of "shipping information" since we don't actually have shipping in core.